### ISSUE TITLE
Change dimension variables in LinearTransform model to integer data type

### ISSUE THREAD

**Issue by chrbertsch** [CATIA-Systems/Test-FMUs#13]
In the modeldescription,xml file of the LinearTransoform model (
https://github.com/CATIA-Systems/Test-FMUs/blob/93110f26d2e18212d4ebd294e46d42719fae2736/LinearTransform/FMI3.xml#L24), the variables `m `and `n `are declared as `Float64` variables. 
Shouldn't they have an integer data type?

**Comment by t-sommer** [comment]
> Shouldn't they have an integer data type?

Yes. It's still a work in progress. Currently the dimensions are fixed.
